mkbootimg/unpackbootimg.c

```diff
@@ -74,6 +74,18 @@ int main(int argc, char** argv)
     if (filename == NULL) {
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit